projects/storefrontlib/shared/components/form/form-errors/form-errors.component.spec.ts

```diff
@@ -13,15 +17,31 @@ describe('FormErrors', () => {
   let component: FormErrorsComponent;
   let fixture: ComponentFixture<FormErrorsComponent>;
   let control: UntypedFormControl;
+  let featureConfigService: jasmine.SpyObj<FeatureConfigService>;
 
   const getContent = () => fixture.debugElement.nativeElement.innerText;
 
   beforeEach(
     waitForAsync(() => {
+      const featureConfigServiceMock = jasmine.createSpyObj(
+        'FeatureConfigService',
+        ['isEnabled']
+      );
```

Comment:
[Nitpick]
It lacks type safety. if ever in the future we'd rename the method, this place will silently fail to rename.
What about `const featureConfigServiceMock: Partial<FeatureConfigService> = ...`?

Reply:
Thanks, changed